backend/yugiohcardrarity/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+
+class YugiohCardRarity(models.Model):
+    rarity = models.CharField(
+        max_length=100,
+        unique=True,
+    )
+
```

Comment:
- [ ] Add rarity `code` field

Exemplary set and code:
`Phantom nightmare` - `PHNI`